infra/build_specified_commit.py

```diff
@@ -76,15 +70,46 @@ def detect_main_repo_from_docker(project_name, example_commit, src_dir='/src'):
   Returns:
     The repo's origin, the repo's name
   """
+  if not helper.check_project_exists(project_name):
+    return None, None
   helper.build_image_impl(project_name)
   docker_image_name = 'gcr.io/oss-fuzz/' + project_name
   command_to_run = [
-      'docker', 'run', '--rm', '-i', '-t', docker_image_name, 'python3',
+      'docker', 'run', '--rm', '-t', docker_image_name, 'python3',
       os.path.join(src_dir, 'detect_repo.py'), '--src_dir', src_dir,
       '--example_commit', example_commit
   ]
   out, _ = execute(command_to_run)
+  match = re.search(r'\bDetected repo: ([^ ]+) ([^ ]+)', out.rstrip())
+  if match and match.group(1) and match.group(2):
+    return match.group(1), match.group(2).rstrip()
```

Comment:
nit: is this `rstrip()` needed, given that you also do `rstrip()` on the `out` two lines above?

Reply:
No it is not.